mines/beta_set[] agent_beta
